Desktop portfolio

NelsonAraujo93 opened the pull request:
In this pull request i acomplished the following activities:

- Linters Corrections.
- Add content to index.html file for desktop version.
- Add css rules in styles.css for Desktop responsiveness.
- Add assets to enhance web UI.
- Add hover actions to work section.

Commit:
Add mediaqueries to desktop screens & changes in index.html for enhance mediaqueries

Commit:
Add icons in assets folder

Commit:
Add icons for the footer in assets folder

Commit:
Add contact desktop css rules & images

Commit:
Add css rules for About desktop section & assets

Commit:
Add css rules for works desktop & assets

Commit:
Add css rules for footer and test Linters

Commit:
Merge branch 'main' into desktop-portfolio

Commit:
Conflict solving

Commit:
Fix a textarea error

Commit:
Css corrections Linters

Commit:
Add new rule in h3 work-card

Debas-31 commented on styles.css:
```diff
@@ -542,3 +580,484 @@ section.footer-m {
+  .mobile-nav {
+    display: none;
+  }
+
```
- [ ] Good job adding a media query to your project of this milestone :+1: , but according to the project requirements, you should use a number between (768...1440)px instead of 760 px as the breakpoint between the desktop layout and another layout.

Debas-31 reviewed (changes requested):
> ## Hi @NelsonAraujo93  :wave:
# Status: Required Change :recycle:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media2.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights 
- Linters test without any error :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- PR title and summary :heavy_check_mark: 
- Commit message :heavy_check_mark:
- Figma template usage :heavy_check_mark: 
- Overall UI design :heavy_check_mark: 
## Required Changes ♻️

- [ ]  Well done adding a README to make a professional readme file :+1: , but something to modify based on Microverse requirements. Kindly customize
  - The README file you used in the previous project.
  - Add the missing sections in your README file. Here is the [Microverse README](https://github.com/microverseinc/readme-template) file template. 
  - Remove the Microverse logo from your README file, or if you have your logo, you can use it.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Update README.md

Commit:
Change MediaQuerie Screen from 760px to 768px

Commit:
Merge branch 'desktop-portfolio' of https://github.com/NelsonAraujo93/MyMicroversePortfolio into desktop-portfolio

NelsonAraujo93 commented:
As the first code review requested i followed the required changes:

- Modify Read.me
- Change MediaQueries css rule from 760px to 768px

rskinnerc commented on README.md:
```diff
@@ -27,11 +27,11 @@
-> Describe your project in 1 or 2 sentences.
-
-**Hello world Microverse-Nelson** is an activity created in the Module 1: Second day of my Microverse classes
+> Description
```
♻️ Great Job implementing the updated version of the README.md file. I will make your repo look more professional. However, there are still a few things you need to work on. As required by the previous reviewer, this template has different sections that help document your work properly. The only optional section is the _FAQ_

- [ ] Kindly add the missing sections as per the README file instructions. Remember the only optional one is FAQ. Even though you are not using some of them, like the Live Demo, you could indicate that this will be added soon.

rskinnerc commented on styles.css:
```diff
@@ -542,3 +580,484 @@ section.footer-m {
   background-size: cover;
 }
+
+@media screen and (min-width: 768px) {
```
⭐ Good job changing the min breakpoint to `768px` as required!

rskinnerc reviewed (changes requested):
### Hi @NelsonAraujo93 👋👋👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![giphy](https://user-images.githubusercontent.com/26911718/206719084-8057aec2-e8c7-412d-8804-a3a1d8005762.gif)


 ## Highlights
- Nice work resembling the Figma Template
- Great! You implemented some js interactivity. I appreciate that.
- You added an extra screen width breakpoint. Nice, it would help to maintain the styling in wider screens

 ## Required Changes ♻️
_Check the comments under the review._

Please update the README.md file as per my comment below. You did a great job so far. I just want your repo to look even more professional. Making this a habit will take you to the next level! Making those small changes would be enough for your project to be approved next review. 💪💪💪

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Commit:
Change Read.me file

I did complete the requirements the reviwers asked me for to complete my project.

DuaneDave reviewed (approved):
Hi @NelsonAraujo93 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Meaningful commit history 💯 
✅ Nice UI 💯 
✅ Professional PR summary 💯 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
